app.js

```diff
@@ -1,32 +1,218 @@
-    // NOTE: Weight in JSON file is in lbs, height in inches.
-
-
-    // Generate Tiles for each Dino in Array
-  
-        // Add tiles to DOM
-
-    // Remove form from screen
-
-
-// On button click, prepare and display infographic
+const HUMAN_INDEX = 4;
+const rawDinos = [
+  {
+    species: "Triceratops",
+    weight: 13000,
+    height: 114,
+    diet: "herbavor",
+    where: "North America",
+    when: "Late Cretaceous",
+    fact: "First discovered in 1889 by Othniel Charles Marsh",
+  },
+  {
+    species: "Tyrannosaurus Rex",
+    weight: 11905,
+    height: 144,
+    diet: "carnivor",
+    where: "North America",
+    when: "Late Cretaceous",
+    fact: "The largest known skull measures in at 5 feet long.",
+  },
+  {
+    species: "Anklyosaurus",
+    weight: 10500,
+    height: 55,
+    diet: "herbavor",
+    where: "North America",
+    when: "Late Cretaceous",
+    fact: "Anklyosaurus survived for approximately 135 million years.",
+  },
+  {
+    species: "Brachiosaurus",
+    weight: 70000,
+    height: "372",
+    diet: "herbavor",
+    where: "North America",
+    when: "Late Jurasic",
+    fact: "An asteroid was named 9954 Brachiosaurus in 1991.",
+  },
+  {
+    species: "Stegosaurus",
+    weight: 11600,
+    height: 79,
+    diet: "herbavor",
+    where: "North America, Europe, Asia",
+    when: "Late Jurasic to Early Cretaceous",
+    fact: "The Stegosaurus had between 17 and 22 seperate places and flat spines.",
+  },
+  {
+    species: "Elasmosaurus",
+    weight: 16000,
+    height: 59,
+    diet: "carnivor",
+    where: "North America",
+    when: "Late Cretaceous",
+    fact: "Elasmosaurus was a marine reptile first discovered in Kansas.",
+  },
+  {
+    species: "Pteranodon",
+    weight: 44,
+    height: 20,
+    diet: "carnivor",
+    where: "North America",
+    when: "Late Cretaceous",
+    fact: "Actually a flying reptile, the Pteranodon is not a dinosaur.",
+  },
+  {
+    species: "Pigeon",
+    weight: 0.5,
+    height: 9,
+    diet: "herbavor",
+    where: "World Wide",
+    when: "Holocene",
+    fact: "All birds are living dinosaurs.",
+  },
+];
+
+// Create Dino Constructor
+function Dino(dinoInfo) {
+  this.species = dinoInfo.species;
+  this.weight = dinoInfo.weight;
+  this.height = dinoInfo.height;
+  this.diet = dinoInfo.diet;
+  this.where = dinoInfo.where;
+  this.when = dinoInfo.when;
+  this.fact = dinoInfo.fact;
+  this.url = `images/${this.species.toLowerCase()}.png`;
+  this.comparedDetails = dinoInfo.comparedDetails;
+}
+
+// Create Dino Objects
+function createDinoObjects() {
+  const arrayDinos = [];
+  for (const dino of rawDinos) {
+    const heightDiff = compareHeight(dino);
+    const weightDiff = compareWeight(dino);
+    const dietDiff = compareDiet(dino);
+    dino.comparedDetails =
+      dino.species === "Pigeon"
+        ? `<h3>Species: ${dino.species}</h3> <br> 
+           All birds are Dinosaurs.`
+        : `
+      <h3>Species: ${dino.species}</h3> <br>

```

Comment:
reduce some repeat title contents maybe better.

Reply:
can you tell me a bit more so I can follow your suggestion better since I am not sure which part is repetitive here?